testnet.Dockerfile

```diff
@@ -3,7 +3,7 @@ ENV DEBIAN_FRONTEND "noninteractive"
 RUN ln --symbolic --force --no-dereference /usr/share/zoneinfo/EST /etc/localtime && \
 	echo "EST" > /etc/timezone && apt-get update && \
```

Comment:
btw, western electric won't tell you this, but there is a world outside of EST

Reply:
we were just throwing stuff at the wall / copying to get it to work in docker